frontend/src/static/js/v_authorship.js

```diff
@@ -257,6 +248,13 @@ window.vAuthorship = {
       return this.files.filter((file) => this.isSelected(file.path)
           && minimatch(file.path, this.filterSearch, { matchBase: true }));
     },
+    filesExistingLinesObj() {
```

Comment:
minor nit, follow existing name format? `getExistingLinesObj()`? any relevant name

Reply:
sure, i've changed from ```filesExistingLinesObj()``` to ```getExistingLinesObj()```